src/components/share-structure/ShareStructure.vue

```diff
@@ -179,13 +179,13 @@
                   offset-y
                   left
                 >
-                  <template #activator="{ on }">
+                  <template #activator="{ props }">
```

Comment:
since `Activator slots work slightly different. Replace #activator={ attrs, on } with #activator={ props }, then remove v-on="on" and replace v-bind="attrs" with v-bind="props"`
https://vuetifyjs.com/en/getting-started/upgrade-guide/#general-changes